core/templates/pages/topic-editor-page/topic-editor-page.component.spec.ts

```diff
@@ -221,4 +221,21 @@ describe('Topic editor page', function() {
     routingSpy.and.returnValue('main');
     expect(ctrl.getNavbarText()).toEqual('Topic Editor');
   });
+
+  it('should load topic based on its id on url when undo or redo action' +
+  'is performed', function() {
```

Comment:
Add a space before `is` so that the description has a space between `action` and `is`